src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalPrice = 0;
+  const cost = 40;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const longRental = 7;
+  const mediumRental = 3;
+
+  if (days >= longRental) {
+    totalPrice += cost * days - longTermDiscount;
+
+    return totalPrice;
+  }
+
+  if (days >= mediumRental) {
+    totalPrice += cost * days - shortTermDiscount;
+
+    return totalPrice;
+  }
+
+  totalPrice += cost * days;
```

Comment:
This line correctly calculates the total price without any discount for rentals less than 3 days.